### PR TITLE
fixing the bugs in the badges and the UI for admin and profile

### PR DESCRIPTION
**Pull request by BrunoNicholas**
Hello there, thank you for your time, please help review this PR and the repo for the standards before other updates.

### CODE REVIEW

**Comment by kalsmic** [comment]
Install the pytest-cov and add it to your requirements.txt to clear the error on travis

**Comment by kalsmic** [comment]
- Update the link to your Heroku, the one the README is not working. It gives a 404 error. It will be nice if you put it as "https://ireporter-v01.herokuapp.com/api/v1/"
- When writing commit messages, use present tense and try to use commit messages that reflect the change you have made.

**Comment by BrunoNicholas** [comment]
Thanks, @kalsmic for the feedback, it is very helpful.
The links and the badges are coming up slowly

File: README.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
-![Build Status](https://travis-ci.org/BrunoNicholas/iReporter.svg?branch=Develope) [![Coverage Status](https://coveralls.io/repos/github/BrunoNicholas/iReporter/badge.svg?branch=Develope)](https://coveralls.io/github/BrunoNicholas/iReporter?branch=Develope)
+[![Build Status](https://travis-ci.org/BrunoNicholas/iReporter.svg?branch=Develope)](https://travis-ci.org/BrunoNicholas/iReporter)
+
+(https://coveralls.io/repos/github/BrunoNicholas/iReporter/badge.svg?branch=Develope)](https://coveralls.io/github/BrunoNicholas/iReporter?branch=Develope)
```
Comment (kalsmic) [inline README.md]: Correct the syntax for the coverage badge. Remove the newline space `[![Coverage Status]` and ` (https://coveralls.io/repos/github/BrunoNicholas/iReporter/badge.svg?branch=Develope)](https://coveralls.io/github/BrunoNicholas/iReporter?branch=Develope)`

Comment (kalsmic) [reply README.md]: The correct way is to go to coveralls and click on embed and copy the markdown

Comment (kalsmic) [reply README.md]: coveralls

Comment (BrunoNicholas) [reply README.md]: Thanks so much, @kalsmic , I was not minding much about the markdown format but I have read about it and it makes much sense

**Pull request merged**